CONTRIBUTING.md

```diff
@@ -5,3 +5,55 @@ We love improvements to our tools! EDGI has general [guidelines for contributing
 ## Submitting Web Monitoring Issues
 
 Issues that are project-wide, or relate heavily to the interaction between different components, should be added to our [Web Monitoring issue queue](https://github.com/edgi-govdata-archiving/web-monitoring/issues). Component-specific issues should be added to their respective repository.
+
+The following are recommend code styling and best practices for all the Web Monitoring repositories. We also have some best practices specifically for our [web-monitoring-ui](https://github.com/edgi-govdata-archiving/web-monitoring-ui/blob/master/CONTRIBUTING.md) repo.
```

Comment:
Seems like this should be *after* the “Code Style / Best Practices” heading, no? At the very least, it’s a bit confusing for it to be under the “Submitting Issues” heading.

Reply:
Agreed.